src/NewTools-DocumentBrowser/MicDocumentBrowserStyleContributor.class.st

```diff
@@ -0,0 +1,16 @@
+"
+A style contributor for the document browser, it adds specific styles to be used by `MicDocumentBrowser` in the context of the `StPharoApplication`.
+"
+Class {
+	#name : #MicDocumentBrowserStyleContributor,
```

Comment:
I do not see where this class is used?

Reply:
is a contributor which is the way a tool part of new-tools has to use to add tool-specific styles to  StPharoApplication